app-editors/vis/vis-0.9-r1.ebuild

```diff
@@ -3,18 +3,22 @@
 
 EAPI=8
 
-MY_PTV=0.5
-LUA_COMPAT=( lua5-2 lua5-3 lua5-4 )
+VIS_TEST_COMMIT="783b7ef67aa360f0b9bd44fa5ea47e644bc49d69"
```

Comment:
Would be better to have a `mv` done in `src_prepare` instead of a custom non-upstreamed commit.

Also would need a referenced bug.

Reply:
Ah woups, github UI changes shenanigans… I thought this commit wasn't in master (it is) and you bumped it.